Dockerfile

```diff
@@ -1,13 +1,25 @@
 # Mozilla Kinto server
-FROM python:3.5
+FROM stackbrew/debian:sid
+MAINTAINER Storage Team irc://irc.freenode.net/#kinto
+
 ADD kinto /code/kinto
-ADD README.rst /code/
-ADD CHANGELOG.rst /code/
-ADD CONTRIBUTORS.rst /code/
-ADD setup.py /code/
-ADD MANIFEST.in /code/
-RUN pip install cliquet[postgresql,monitoring]
-RUN pip install -e /code
+ADD CHANGELOG.rst README.rst CONTRIBUTORS.rst setup.py MANIFEST.in /code/
 ENV KINTO_INI /etc/kinto/kinto.ini
-RUN kinto --ini $KINTO_INI --backend=memory init
-CMD kinto --ini $KINTO_INI migrate && kinto --ini $KINTO_INI start
+
+# Install build dependencies, build the virtualenv and remove build
+# dependencies all at once to build a small image.
+RUN \
+    apt-get update; \
+    apt-get install -y python3 python3-pip python3-venv git build-essential make; \
+    apt-get install -y python3-dev libssl-dev libffi-dev libpq5 libpq-dev; \
+    python3 -m venv /home/kinto; \
+    /home/kinto/bin/pip install -U pip; \
+    /home/kinto/bin/pip install cliquet[postgresql,monitoring]; \
+    /home/kinto/bin/pip install -e /code; \
+    /home/kinto/bin/kinto --ini $KINTO_INI --backend=memory init; \
+    apt-get remove -y -qq git build-essential git make python3-pip python3-venv libssl-dev libffi-dev libpq-dev python3-dev; \
```

Comment:
Add a comment like: `Uninstall from container every lib that are used for installation only`